core/rawdb/accessors_chain.go

```diff
@@ -617,7 +617,7 @@ func DeleteWorkObjectHeader(db ethdb.KeyValueWriter, hash common.Hash, woType in
 }
 
 // ReadWorkObject retreive's the work object stored in hash.
-func ReadWorkObject(db ethdb.Reader, hash common.Hash, woType int) *types.WorkObject {
+func ReadWorkObject(db ethdb.Reader, hash common.Hash, woType types.WorkObjectView) *types.WorkObject {
```

Comment:
same here as well, cant have constant like that

Reply:
Same comment, this isn't a constant.

Reply:
This also applies to all the non-replied to comments below.